usmqe_tests/api/grafana/test_volume_dashboard.py

```diff
@@ -0,0 +1,107 @@
+"""
+REST API test suite - Grafana dashboard volume-dashboard
+"""
+
+import pytest
+from usmqe.api.grafanaapi import grafanaapi
+
+
+LOGGER = pytest.get_logger('volume_dashboard', module=True)
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+``GRAFANA`` for this file stands for Grafana API url used by tested Tendrl
+server.
+``GRAPHITE`` for this file stands for Graphite API url used by tested Tendrl
+server.
+
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_layout():
```

Comment:
Could you rename this to something like `test_volume_dashboard_layout`? It would make logs more easier to read.